sql/moz-fx-data-shared-prod/mozilla_vpn_derived/all_subscriptions_v1/query.sql

```diff
@@ -178,6 +186,7 @@ fxa_subscriptions AS (
     ) AS pricing_plan,
     -- Stripe default billing grace period is 1 day and Paypal is billed by Stripe
     INTERVAL 1 DAY AS billing_grace_period,
+    [promotion_code] AS promotion_codes,
```

Comment:
using an array because having more than one is possible in the future (but now right now)

Reply:
From the schema it doesn't look like Stripe allows multiple coupons per subscription, so can you elaborate on how multiple promotion codes are possible in the future?  (I'm guessing maybe from other payment providers, or over time as subscriptions are renewed)

Reply:
I asked subplat eng in slack and annasob replied `it can be configured to allow multiple ones but for now you can only use one at checkout`. I believe the data model for mulitple coupons is that they are attached to invoices, instead of subscriptions.